web/src/components/Alert.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { Alert, Button } from '@cmsgov/design-system';
+
+import { getTempMessages } from '../redux/reducers/alerts';
+
+const TempAlert = ({ tempMessages }) => {
+  const [tempMsgState, setTempMsgState] = useState(tempMessages);
+
+  if (!tempMessages) {
+    return null;
+  }
+
+  const removeMsg = i => {
+    setTempMsgState([
+      ...tempMsgState.slice(0, i),
+      ...tempMsgState.slice(i + 1)
+    ]);
+  };
+
+  return (
+    <div>
+      {tempMsgState.map((value, index) => (
+        <Alert
+          key={`tempMsg-${index + 1}`}
+          className="ds-u-margin-y--2"
+          variation={value.variation}
+        >
+          <div className="tempMessage">
+            <div>{value.message}</div>
+            <div>
+              <Button onClick={() => removeMsg(index)} className="tempMsgBtn">
+                X
```

Comment:
We should probably use the design system `CloseIcon` here
https://design.cms.gov/components/icon/?theme=core

Reply:
ooooh, good call